Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine
```

Comment:
We should use a tagged, stable version of alpine such as `3.10`.

Reply:
Done